tests/unit/test_ncbi_source.py

```diff
@@ -661,21 +661,6 @@ def test_prkrap1(check_resp_single_record, ncbi, prkrap1):
     check_resp_single_record(resp, prkrap1, MatchType.XREF)
 
 
-def test_mhb(check_resp_single_record, ncbi, mhb):
-    """Test that MHB normalizes to correct gene concept."""
-    # Concept ID
-    resp = ncbi.search("NCBIgene:619511")
-    check_resp_single_record(resp, mhb, MatchType.CONCEPT_ID)
-
-    # Symbol
-    resp = ncbi.search("MHB")
-    check_resp_single_record(resp, mhb, MatchType.SYMBOL)
-
-    # associated_with
-    resp = ncbi.search("OMIM:255160")
-    check_resp_single_record(resp, mhb, MatchType.ASSOCIATED_WITH)
```

Comment:
This got discontinued. We already have tests for discontinued symbols / genes with previous symbols, so removed this test.